### PR TITLE
Fix wrong georeference results layouts

### LINKED ISSUE #4236: Wrong layout/icons on new georeference results modal
use new icons and layout:

- [x] no georeferenced data (points)
screen shot 2015-07-17 at 15 57 22

- [x] no georeferenced data (polygons)
screen shot 2015-07-17 at 15 57 28

- [x] geocoding succesful - default
screen shot 2015-07-17 at 15 57 53

- [x] geocoding successful - no cost
screen shot 2015-07-17 at 15 57 41

- [x] geocoding successful - cost
screen shot 2015-07-17 at 15 57 47

### PR DESCRIPTION
**Pull request by viddo**
Fixes #4236 

Note that I'm merging into the branch of an existing PR, since I branched off that one to have the latest changes that were affected, I'll release both once this is OK'd.

Regarding the changes, following the styles in the sketch file + reusing the current header styles to be consistent with what already exists (and since header styles are planned to be consolidated in #4175 anyway).

Some screenshots:
screen shot 2015-07-20 at 11 31 50
screen shot 2015-07-20 at 11 45 28
screen shot 2015-07-20 at 11 48 02
screen shot 2015-07-20 at 11 50 07
screen shot 2015-07-20 at 12 03 36

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5151/))

**Comment by josecruz** [comment]
Georeference cost confirmation modal:
screen shot 2015-07-20 at 12 48 29 pm

**Comment by viddo** [comment]
Good catch! Was already done in the georeference modal (these modals were for the results, that's opened through background importer). But updated the text styles in the last 2 commits to match the results ones too: screen shot 2015-07-20 at 13 09 28

**Comment by viddo** [comment]
@xavijam can you please review? I think you're the best one since you did the orignal results templates. I could remove a lot of unused variables, and consolidated two the existing templates, since price info is only displayed inline if it's available.

cc @josecruz for visuals

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5154/))

**Comment by alonsogarciapablo** [comment]
Hey @viddo, I will review this.

**Comment by josecruz** [comment]
:+1: for visuals

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5155/))

File: lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_success_details.jst.ejs [hunk 1]
```diff
@@ -1,43 +1,63 @@
...
+    <div class="Georeference-resultsBodyIcon LayoutIcon <%- price > 0 ? 'LayoutIcon--warning' :  'LayoutIcon--positive' %>">
+      <span class="iconFont iconFont-Dollar iconFont--super"></span>
+    </div>
+    <div class="Georeference-resultsBodyTexts">
+      <% if (price > 0) { %>
+        <p class="DefaultTitle">
+          <strong>$<%= price / 100 %></strong> will be charged to your account
+        </p>
+        <p class="DefaultParagraph DefaultParagraph--tertiary">
+          You have consumed all your credits during this billing cycle. (price is $<%- blockPrice / 100  %>/1,000 extra credits)
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_success_details.jst.ejs]: Period before parentheses feels a bit weird for me. I would either move the period after the closing parenthesis, remove the parentheses or change it to: `...cycle. (Price is $<%- blockPrice / 100  %>/1,000 extra credits.)`.

File: lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_result_details_view.js [hunk 2]
```diff
@@ -19,21 +19,19 @@ module.exports = BaseDialog.extend({
   },
 
   render_content: function() {
-    var processedRows = this.model.get('processed_rows') || 0;
+    var error = this.model.get('error') || {};
+
+    var processedRows = this.model.get('processed_rows') || 0;
     var processableRows = this.model.get('processable_rows') || 0;
     var realRows = this.model.get('real_rows') || 0;
-    var googleUser = this.user.featureEnabled('google_maps');
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_result_details_view.js]: Nice cleanup!

File: lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_success_details.jst.ejs (hunk 1 above)
Comment (viddo) [inline lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_success_details.jst.ejs]: Yeah, sorry my bad, copy-paste from the sketch file… I think changing to a comma and removing the parenthesis makes it feel more aligned with exiting copy and standard grammar.

I.e. _You have consumed all your credits during this billing cycle, price is $123/1,000 extra credits._

cc @josecruz

File: lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_success_details.jst.ejs [hunk 3]
```diff
@@ -1,43 +1,63 @@
...
+          Unsuccessful rows don't count against your quota, so we encourage you to take a look and try again.
+        <% }%>
+      </p>
+    </div>
   </div>
 <% } %>
-<div class="BackgroundPollingDetails-footer">
-  <button class="Button Button--secondary Button--inline BackgroundPollingDetails-footerButton cancel">
+
+<% if (!googleUser && hasPrice) { %>
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_success_details.jst.ejs]: Can `(processableRows > realRows)` (in [this line](https://github.com/CartoDB/cartodb/pull/4567/files#diff-0c2ab59b48378902564abe0e30bad04dL21)) and `(!googleUser && hasPrice)` can be true at the same time?

Comment (viddo) [inline lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_success_details.jst.ejs]: Yes, they're not dependant on each other. One has to do with billing (geocodings that are going through Nokia Here), the other is related to geocoding failing some rows for whatever reason. E.g. as seen in 3rd screenshot:

screen shot 2015-07-20 at 11 48 02

Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb/common/background_polling/views/geocodings/geocoding_success_details.jst.ejs]: Perfect, just wanted to make sure.

**Comment by alonsogarciapablo** [comment]
I assume @viddo has done some exhaustive QA of the look and feel of these modals, so :+1:.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5164/))

**Pull request merged**